src/rust/engine/process_execution/src/remote_tests.rs

```diff
@@ -2846,25 +2821,15 @@ pub(crate) fn cat_roland_request() -> MultiPlatformProcess {
     jdk_home: None,
     target_platform: PlatformConstraint::None,
     is_nailgunnable: false,
+    execution_slot_variable: None,
   };
   req.into()
 }
 
 pub(crate) fn echo_roland_request() -> MultiPlatformProcess {
-  let req = Process {
-    argv: owned_string_vec(&["/bin/echo", "meoooow"]),
-    env: BTreeMap::new(),
-    working_directory: None,
-    input_files: EMPTY_DIGEST,
-    output_files: BTreeSet::new(),
-    output_directories: BTreeSet::new(),
-    timeout: one_second(),
-    description: "unleash a roaring meow".to_string(),
-    append_only_caches: BTreeMap::new(),
-    jdk_home: None,
-    target_platform: PlatformConstraint::None,
-    is_nailgunnable: false,
-  };
+  let mut req = Process::new(owned_string_vec(&["/bin/echo", "meoooow"]));
+  req.timeout = one_second();
+  req.description = "unleash a roaring meow".to_string();
```

Comment:
I had started to add the builder pattern to Process for this usecase but... what you're doing here is probably just as easy? https://github.com/pantsbuild/pants/blob/3d79217c57bc5a0a71a01aac9a9cae41a73a1994/src/rust/engine/process_execution/src/lib.rs#L263-L271

Reply:
> I had started to add the builder pattern to Process for this usecase but... what you're doing here is probably just as easy?
> 
> https://github.com/pantsbuild/pants/blob/3d79217c57bc5a0a71a01aac9a9cae41a73a1994/src/rust/engine/process_execution/src/lib.rs#L263-L271

It looks like there's a half-complete builder pattern in the codebase currently. It would be nice to either extend that to every option we care about, or get rid of it and just mutate member variables on `Process` directly, but either way, it's out of the scope of this PR.